chunks/approximate_methods.ipynb

```diff
@@ -0,0 +1,385 @@
+    "<td><img src=\"../graphics/approx_des_ngmix_xi.png\" width=70%></td>\n",
+    "</tr></table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Example: Cosmic Shear\n",
+    "\n",
+    "* The \"data\" are automatically-measured galaxy shapes, whose tiny distortions are due to weak gravitational lensing by a complex framework of large scale dark matter structure pervading the entire universe between those background galaxies and us"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Example: Cosmic Shear\n",
+    "\n",
+    "* Rather then hierarchically model this structure, with cosmological hyper-parameters, we compress the shape data into \"correlation function\" summary statistics _which can be directly predicted from a cosmological model_\n",
+    "\n",
+    "\n",
+    "* Issue: the exact form, and the multi-variate width, of the sampling distribution for the correlation function data is unknown (and dependent on cosmology)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Unknown Likelihood Function\n",
+    "\n",
+    "What should we do if we don't know (or don't trust our model for) the sampling distribution of our data?\n",
+    "\n",
+    "\n",
+    "1. Model it in a flexible way (as in the [\"model-free models\"](modelfreemodels.ipynb) chunk)\n",
+    "\n",
+    "2. If we can generate mock data anyway - try **ABC** "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Approximate Bayesian Computation\n",
+    "\n",
+    "ABC is a family of sampling methods all based on the idea that if we can generate a mock dataset that is _close to_ the observed data, then the parameters of that model are _approximately_ a plausible draw from the posterior PDF\n",
+    "\n",
+    "> The closer we get, the better the approximation\n",
+    "\n",
+    "We are in this situation when the model is  too complex to enable a likelihood to be evaluated (or even written down). \n",
+    "\n",
+    "> Examples: Human ancestry given DNA sequence data. Galaxy evolution given survey data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### The Basic ABC Algorithm\n",
+    "\n",
+    "Suppose we have a dataset $d$, and a generative model $H$ with parameters $\\theta$:\n",
+    "\n",
+    "* Draw $\\theta$ from the prior PDF $P(\\theta|H)$\n",
+    "\n",
+    "* Generate a mock dataset $d'$\n",
+    "\n",
+    "* Compute the \"distance\" between the observed and mock datasets, $\\rho(d,d')$\n",
```

Comment:
It might be good to say something more about our freedom to choose the distance function. The ideal one is the likelihood itself, which is a useful link to make.